tests/release/pipelines/fbc_release.go

```diff
@@ -214,7 +214,7 @@ func assertBuildPipelineRunSucceeded(devFw framework.Framework, devNamespace, ma
 	// Create a ticker that ticks every 3 minutes
 	ticker := time.NewTicker(3 * time.Minute)
 	// Schedule the stop of the ticker after 5 minutes
```

Comment:
this comment doesn't correspond with what's in the code below - btw what's the motivation for this change?

Reply:
The build pipelineRun need a longer time than before in a recent test. The access token expired and test case failed. So, the PR is to fix it.